federation/pkg/federation-controller/util/cluster_util.go

```diff
@@ -146,3 +149,55 @@ func GetClientsetForCluster(cluster *federation_v1beta1.Cluster) (*fedclientset.
 	}
 	return nil, err
 }
+
+// Returns a bool to indicate if the object should be forwarded to a cluster
+func SendToCluster(cluster *federation_v1beta1.Cluster, objMeta metav1.ObjectMeta) bool {
```

Comment:
This should take annotations (map[string]string) as an argument. It does not require the whole ObjectMeta.
That will make the test simpler (wont have to create ObjectMeta in the test).